src/randoop/main/UniversalDriverHandler.java

```diff
@@ -31,6 +31,7 @@ public boolean handle(String[] args) throws RandoopTextuiException {
 
     Options parsedArgs = new Options(usage_synopsis,
         WriteModelCheckerDriver.class);
+    @SuppressWarnings("unused")
     String non_options[] = null;
```

Comment:
This isn't your fault, but could you remove the initialization?  It isn't necessary because of the assignment in a following line.